### DISCUSSION TITLE
Pointcloud integration

### DISCUSSION THREAD

**Discussion by giaplesg** [TerriaJS/terriajs#5265] [Q&A] [answered]
First of all, great work with terriajs its absolutelly amazing keep it up guys!!!
I have a pointcloud from some regions and i would like to import them at the globe, such as cesium does with these types of files. Is there any way in terria that i can implement this?
The types of files can be .laz or .ply or any other supported (binary) format.

**Comment by hardreddata** [comment, marked as answer]
Hi @giaplesg 

The simplest way would probably be to get a trial Cesium ion account and upload the model there.

You can then access via the Asset ID and https://docs.terria.io/guide/connecting-to-data/catalog-type-details/3d-tiles/ using something like this catalogue item.

```
{
   "name":"My Model",
   "type":"3d-tiles",
   "description":"My description",
   "ionAssetId":12345,
   "ionAccessToken": abc123
}
```

**Reply by jasprx** [reply]
Hello @RussellGrew 
Is is possible to show my own 3d tilesets that I've made? I have few of them and I'd like to show them in terria just like I can do it in my Cesium app. Can you elaborate please?
Thank you in advance.

**Reply by hardreddata** [reply]
Hi @wonszopaionk 

You can absolutely do this.

The code snippet I shared above is for a Terria catalog. A more complete example is https://raw.githubusercontent.com/TerriaJS/terriajs/master/wwwroot/test/init/cesium3dtiles.json

Take a look at https://docs.terria.io/guide/ and specifically https://docs.terria.io/guide/connecting-to-data/catalog-type-details/3d-tiles/

Cheers.

**Reply by jasprx** [reply]
Great, thank you! 
I've been loking there into the docs, but I was unsure whether it would be possible, because I wasn't able to show the dataset in my terria. Turns out that I have some issues with CORS and http request returning not entire tileset.json file. :/ 
Nontheless thank you for the response.
